README.md

```diff
@@ -202,6 +202,18 @@ You can generate a PDF or an HTML copy of this guide using
       )
     end
     ```
+    
+* When the purpose of one or more parameters to a method is opaque, introduce
+  an inline explanatory variable to match the method signature.
+
+  ```Ruby
+  
+  # good (avoid mysterious booleans)
+  person.save(validate = false)
+  
+  # good (distinguish between ordered arguments of the same type)
+  FooReader.new.read(max_retries = 0, max_bytes = 255)
+  ```
```

Comment:
I think it could assist to have a `# bad` example, even if its copypasta of someone ignominious code